nf_core/modules/install.py

```diff
@@ -0,0 +1,173 @@
+import os
+import sys
+import json
+import questionary
+import logging
+
+import nf_core.utils
+
+from .modules_command import ModuleCommand
+from .module_utils import get_module_git_log
+
+log = logging.getLogger(__name__)
+
+
+class ModuleInstall(ModuleCommand):
+    def __init__(self, pipeline_dir, module=None, force=False, latest=False, sha=None):
+        super().__init__(pipeline_dir)
+        self.force = force
+        self.latest = latest
+        self.sha = sha
+        module = module
+
+    def install(self, module):
+        if self.repo_type == "modules":
+            log.error("You cannot install a module in a clone of nf-core/modules")
+            sys.exit(1)
```

Comment:
Can we raise an exception instead and catch it in the cli code? I prefer not to have exits anywhere but there if possible.

Reply:
Other checks just return false. So that would also be fine.